temporal-serviceclient/src/main/java/io/temporal/serviceclient/StatusUtils.java

```diff
@@ -87,7 +87,7 @@ public static <T extends GeneratedMessageV3> StatusRuntimeException newException
    * This method does exactly what {@link Any#pack(Message)} does. But it doesn't go into reflection
    * to fetch the {@code descriptor}, which allows us to avoid a bunch of Graal reflection configs.
    */
-  private static <T extends GeneratedMessageV3> Any packAny(
+  public static <T extends GeneratedMessageV3> Any packAny(
```

Comment:
Can we move this to internal if it needs to be public?

Reply:
Like so https://github.com/temporalio/sdk-java/pull/2199/commits/caf4fa1e18b703ddf3fc20b6b952425b80ad74cc ?

Reply:
Yeah, this is just meant to be an internal helper method not something we want users of the SDK to use